src/components/screens/transactionDetails/transactionVotes.js

```diff
@@ -27,22 +23,13 @@ const transactionVotes = ({ t, transaction, delegates }) => {
           {`${t('Votes')} (${votes.length})`}
         </span>
         <div className={`${styles.votesContainer} ${styles.added} tx-added-votes`}>
-          {votes.map(vote => (
-            <Link
-              key={vote.delegateAddress}
-              to={`${accountPath}?address=${vote.delegateAddress}`}
-              className={`${styles.voteTag} voter-address`}
-            >
-              <span className={styles.username}>
-                {delegates[vote.delegateAddress]
-                  ? delegates[vote.delegateAddress].username
-                  : vote.delegateAddress
-                }
-              </span>
-              <span className={styles.voteAmount}>
-                <LiskAmount val={vote.amount} token={tokenMap.LSK.key} />
-              </span>
-            </Link>
+          {votes.map((vote, i) => (
+            <VoteItem
+              key={i}
```

Comment:
I would recommend to use an id instead of index as it have negative impact on react. I suggest to use `vote.delegateAddress`.
There is documentation about this [here](https://reactjs.org/docs/lists-and-keys.html)